app/renderer/components/tabs/tabs.js

```diff
@@ -4,67 +4,69 @@
 
 const React = require('react')
 const ReactDOM = require('react-dom')
+const Immutable = require('immutable')
 
 // Components
-const ImmutableComponent = require('../immutableComponent')
+const ReduxComponent = require('../reduxComponent')
 const LongPressButton = require('../common/longPressButton')
 const Tab = require('./tab')
 
 // Actions
 const appActions = require('../../../../js/actions/appActions')
 const windowActions = require('../../../../js/actions/windowActions')
 
+// State
+const windowState = require('../../../common/state/windowState')
+const tabContentState = require('../../../common/state/tabContentState')
+
 // Constants
 const dragTypes = require('../../../../js/constants/dragTypes')
+const settings = require('../../../../js/constants/settings')
 
 // Utils
 const cx = require('../../../../js/lib/classSet')
 const contextMenus = require('../../../../js/contextMenus')
-const {getCurrentWindowId} = require('../../currentWindow')
+const {getCurrentWindowId, isFocused} = require('../../currentWindow')
 const dnd = require('../../../../js/dnd')
 const dndData = require('../../../../js/dndData')
+const frameStateUtil = require('../../../../js/state/frameStateUtil')
+const {getSetting} = require('../../../../js/settings')
 
-class Tabs extends ImmutableComponent {
-  constructor () {
-    super()
+class Tabs extends React.Component {
+  constructor (props) {
+    super(props)
     this.onDragOver = this.onDragOver.bind(this)
     this.onDrop = this.onDrop.bind(this)
     this.onPrevPage = this.onPrevPage.bind(this)
     this.onNextPage = this.onNextPage.bind(this)
     this.onNewTabLongPress = this.onNewTabLongPress.bind(this)
-    this.wasNewTabClicked = this.wasNewTabClicked.bind(this)
     this.onMouseLeave = this.onMouseLeave.bind(this)
-    this.onTabClosedWithMouse = this.onTabClosedWithMouse.bind(this)
   }
 
   onMouseLeave () {
+    if (this.props.fixTabWidth) {
+      return
```

Comment:
@cezaraugusto this if was copied from https://github.com/brave/browser-laptop/pull/8396/files#diff-15331dd5e010a9a3d250a1a37d851653L132. If I remove it from a function is working correctly, but I would like to understand why it was there in the first place